regolith/helpers/a_presentationhelper.py

```diff
@@ -119,6 +126,61 @@ def construct_global_ctx(self):
         rc.coll = f"{TARGET_COLL}"
         if not rc.database:
             rc.database = rc.databases[0]["name"]
+
+        # if an expense database is specified using the --expense-db option,
+        if rc.expense_db:
+
+            # check that it's known / find it in rc.databases. Store its index number so we can access its other keys inside rc.databases
+            index = 0
+            db_known = False
+            while index < len(rc.databases):
+                if rc.databases[index]["name"] == rc.expense_db:
+                    db_known = True
+                    break
+                index += 1
+            expense_db_index = index
+
+            # If the database is not known, exit with descriptive error message
+            if not db_known:
+                 raise RuntimeError(
+                    f"WARNING: The expense database provided, {rc.expense_db}, does not exist. "
+                    "Please rerun specifying a known private database. "
```

Comment:
similarly, I would combine these I think. If am a user, I may not know what "specifying a known private database means".  This  sounds a bit like one of those frustrating error messages written by a developer that doesn't really help me as a user to know what I am supposed to do to fix the situation.  Come up with your own (or use this) but I think if I was a user a message something like `f"ERROR: You specified database {rc.expense_db} for the expenses collection when you ran this helper, but I don't find that database specified in your regolithrc.json file.  Please check the spelling when you ran the program and that the database is correctly specified in regolithrc.json."`